packages/cli-e2e/tsconfig.json

```diff
@@ -7,6 +7,7 @@
     "skipLibCheck": true,
     "sourceMap": true,
     "forceConsistentCasingInFileNames": true,
-    "types": ["jest"]
+    "types": ["jest"],
+    "lib": ["dom", "es2017"]
```

Comment:
@louis-bompart, this additional line is required for the `isElementClickable` function